src/main/java/com/auth0/net/VoidRequest.java

```diff
@@ -21,10 +21,4 @@ protected Void parseResponse(Response response) throws Auth0Exception {
         }
         return null;
     }
-
-    @Override
-    public VoidRequest setCustomFields(Map<String, String> customFields) {
```

Comment:
So this is a BC as Public API removal, this is a compile fail for anyone using this?

Reply:
"CustomFields" were supposed to be available only for SignUp methods. Because of how the request classes were inherited, this method ended wrongly in the `VoidRequest` class. No, it wouldn't be a breaking change as there's not a single API method signature that declares a return type of VoidRequest. They all either return an `AuthRequest` (never a `TokenRequest`), `SignUpRequest` (never a `CreateUserRequest`) or a simple `Request` or `Request<T>`.

Reply:
If people were casting that returned type to `VoidRequest` (in the case that that method call was returning a `VoidRequest` class or subclass) and using that method, then yes it would be a breaking change. But that's why we return interfaces rather than classes, to obscure our internals. So I wouldn't consider this a breaking change anyway.